src/Sarif.Converters/StaticDriverVerifierConverter.cs

```diff
@@ -6,17 +6,20 @@
 using System.Diagnostics;
 using System.IO;
 using System.Text;
+using System.Text.RegularExpressions;
 
 namespace Microsoft.CodeAnalysis.Sarif.Converters
 {
     internal class StaticDriverVerifierConverter : ToolFileConverterBase
     {
         private StringBuilder _sb;
+        private Stack<string> _callers;
 
         /// <summary>Initializes a new instance of the <see cref="StaticDriverVerifierConverter"/> class.</summary>
         public StaticDriverVerifierConverter()
         {
             _sb = new StringBuilder();
+            _callers = new Stack<string>();
         }
 
         /// <summary>
```

Comment:
Also I read over the `ProcessSdvDefectStream` method again. Instead of reading bytes from a stream until EOF, and looking for newlines, you could make a `StreamReader` out of the `input` stream, and then just call `ReadLine` on it.